pkg/agent/metrics/prometheus.go

```diff
@@ -20,55 +20,74 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	metricNSAntrea = "antrea"
+	subsystemProxy = "agent"
```

Comment:
out of curiosity, where does the term "proxy" come from here? Is "subsystem proxy" a Prometheus concept?

Reply:
My bad, I just copied the variable in the proxier. Updated